tests/Unit/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.py

```diff
@@ -5,22 +5,20 @@
 
 
```

Comment:
Lump this file into the formatting commit as well. Only one function was actually added here.